scripts/intfutil

```diff
@@ -301,7 +301,10 @@ def appl_db_portchannel_status_get(appl_db, config_db, po_name, status_type, por
         status = portchannel_speed_dict[po_name]
         return status
     if status_type == "vlan":
-        status = "routed"
+        if po_name in combined_int_to_vlan_po_dict.keys():
```

Comment:
Since you've the param default to `None`, it is better to check before getting `keys`. If `None`, return `status `as "`routed`"

Reply:
@prsunny fixed it please check.